### DISCUSSION TITLE
Uploading photos

### DISCUSSION THREAD

**Discussion by leozulfiu** [alangrainger/immich-public-proxy#17] [Feature Requests]
I sometimes make my server open to the public so my friends can upload their photos to a central album. Is it this use case also supported by the proxy?

**Comment by leozulfiu** [comment]
And are there any plans to somehow integrate this project into immicht itself?
For me, it seems the most logical step.

**Reply by alangrainger** [reply]
Immich isn't my project, so it would be up to the Immich devs to integrate something like this. I do think it's a good idea to have a separate viewer container like this, and I make the same suggestion here: https://www.reddit.com/r/selfhosted/comments/1gfhcyf/introducing_immich_public_proxy_safely_share_your/lunrfi3/

**Comment by alangrainger** [comment]
Sorry, but no that's not something which will be added for this project. I won't be adding any features which require Write access to Immich, as the purpose of this project is to abstract away from any risk to your Immich instance.
